Stacey Brosnan/week02/Thursday-jQuery/Library/js/exercises.js

```diff
@@ -0,0 +1,34 @@
+
+const books = [
+  {
+    title: 'The Design of EveryDay Things',
+    author: 'Don Norman',
+    alreadyRead: false
+  },
+  {
+    title: 'The Most Human Human',
+    author: 'Brian Christian',
+    alreadyRead: true
+  }
+];
+
+// - Iterate through the array of books. For each book, create a p element with the book title and author and append it to the page./
+
+const $ul = $('<ul></ul>'); //detached DOM node
+
+
+        for (let i = 0; i < books.length; i++){
```

Comment:
Could also use [.each()](https://api.jquery.com/jQuery.each/) here. For loop also works though!